LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/Nct677X.cs

```diff
@@ -817,11 +817,37 @@ private byte ReadByte(ushort address)
         }
 
         byte page = (byte)(address >> 8);
-        byte index = (byte)(address & 0xFF);
-        Ring0.WriteIoPort(_port + EC_SPACE_PAGE_REGISTER_OFFSET, EC_SPACE_PAGE_SELECT);
+        byte index = (byte)(address & 0xFF);
+
+        //wait for access, access == EC_SPACE_PAGE_SELECT
+        //after 100 retries, abort and force access
+        byte access;
+        int timeout = 100;
+        do
+        {
+            access = Ring0.ReadIoPort(_port + EC_SPACE_PAGE_REGISTER_OFFSET);
+            if (access != EC_SPACE_PAGE_SELECT)
+            {
+                timeout--;
+                System.Threading.Thread.Sleep(1);

```

Comment:
Is there a reason why the pull request is not accepted?

Reply:
Code not formatted yet and I don't think the current timeout is OK. Thread.Sleep(1) is ~16ms on Windows, with 100 tries, this would mean the application could potentially hang up to 1.6s for every time {Read|Write}Byte is executed (quite often).

Reply:
the loop terminates after 500ms.
` DateTime timeout = DateTime.UtcNow.AddMilliseconds(500);
        int retries = 100;`

Reply:
new formatting base on editorconfig and timeout update.